data/formatted/exchanges.json

```diff
@@ -354,8 +354,8 @@
       "status": "no"
     },
     "send_to_bech32m": {
-      "status": "evaluating",
-      "detail": "??"
+      "status": "yes",
+      "detail": "since at least November 2022"
```

Comment:
Sorry, the source file we’re using to generate the site is `data/formatted/support.json`. I’ve also moved your name from the list of the website’s creators to the credit section of the entry.